src/ucp/core/ucp_context.c

```diff
@@ -84,12 +89,19 @@ static ucs_config_field_t ucp_config_table[] = {
    "Estimation of buffer copy bandwidth",
    ucs_offsetof(ucp_config_t, ctx.bcopy_bw), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"ATOMIC_MODE", "device",
+   "Atomic operations synchronization mode.\n"
+   " host   - operations are atomic with respect to the host processor.\n"
```

Comment:
when I think about host, I think about CPU, GPGPU, etc and then the term becomes confusing. It really depends on what memory you operate and where the memory controller resides...This is not a blocker, but I want to make the point